test/Microsoft.VisualStudio.Composition.Tests/XunitExtensions/Mef3DiscoveryTestCaseRunner.cs

```diff
@@ -197,10 +197,12 @@ private static async Task RoundtripCatalogSerializationAsync(ComposableCatalog c
             var ms = new MemoryStream();
             catalogSerialization.SaveAsync(catalog, ms).Wait();
             ms.Position = 0;
-            var deserializedCatalog = await catalogSerialization.LoadAsync(ms, TestUtilities.Resolver);
 
             var before = new StringWriter();
             catalog.ToString(before);
+
+            var deserializedCatalog = await catalogSerialization.LoadAsync(ms, TestUtilities.Resolver);
```

Comment:
Is there some significant reason behind this move? Did the old placement quit working, or are you testing something specifically by moving this?